python/llm/example/GPU/Deepspeed-AutoTP/README.md

```diff
@@ -3,7 +3,9 @@
 This example demonstrates how to run BigDL-LLM optimized low-bit model on multiple [Intel GPUs](../README.md) by leveraging DeepSpeed AutoTP.
 
 ## Requirements
-To run this example with BigDL-LLM on Intel GPUs, we have some recommended requirements for your machine, please refer to [here](../README.md#recommended-requirements) for more information. For this particular example, you will need at least two GPUs on your machine.
+To use Intel GPUs for deep-learning tasks, you should install GPU driver and oneAPI Base Toolkit beforehand. See the [GPU installation guide](https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html) for more details.
```

Comment:
To use Intel GPUs for deep-learning tasks, -> To run this example with BigDL-LLM on Intel GPUs,